### PR TITLE
Geofeature resources migration to composite resource type

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by sblack-usu** [comment]
This ran on beta with no errors

**Comment by sblack-usu** [comment]
approving, though this PR has the same duplicate code problem as the raster PR you're cleaning up.  It can be merged after it's cleaned up

File: hs_core/management/commands/migrate_geofeature_resources.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            gf_res.resource_type = to_resource_type
+            gf_res.content_model = to_resource_type.lower()
+            gf_res.save()
+            # get the converted resource object - CompositeResource
+            comp_res = gf_res.get_content_model()
+
+            # set CoreMetaData object for the composite resource
+            core_meta_obj = CoreMetaData.objects.create()
+            comp_res.content_object = core_meta_obj
+            comp_res.save()
```
Comment (sblack-usu) [inline hs_core/management/commands/migrate_geofeature_resources.py]: This needs to be moved to line 128.  If you save the resource with empty metadata then last_updated will throw an error during the save.  (I moved it in the beta test)

Comment (pkdash) [reply hs_core/management/commands/migrate_geofeature_resources.py]: Yeah this migration code is quite old - the last_updated code is probably newer.

**Pull request merged**